pkg/oci/driver/driver.go

```diff
@@ -96,6 +96,18 @@ func GetConfigPath() string {
 	return filepath.Join(path, "config.yaml")
 }
 
+// GetKubeconfigPath gets the override path of the 'kubeconfig'. This override
+// can be uses to explicitly set the name and location of the kubeconfig file
+// via the OCI_FLEXD_KUBECONFIG environment variable. If this value is not
+// specified then the default GetConfigDirectory mechanism is used.
+func GetKubeconfigPath() string {
+	kcp := os.Getenv("OCI_FLEXD_KUBECONFIG_PATH")
```

Comment:
Technical it's the kubeconfig directory path not the kubeconfig path. Could trip someone up.